ui/admin/app/components/form/group/add-members/index.hbs

```diff
@@ -41,7 +41,11 @@
               {{/if}}
             </Hds::Form::Checkbox::Field>
           </B.Td>
-          <B.Td><code>{{B.data.id}}</code></B.Td>
+          <B.Td>
+            <Hds::Text::Code @tag='code'>
```

Comment:
For these examples, should the tag actually be `code`? It seems to me we used the `code` element for the display we wanted but it wasn't actually semantically code blocks.

Should we just be displaying it as code without the code tags?

Reply:
I see your point. In that case, we can just use `Hds::Text::Code` as is (`span`) since we really just need it for the display.

Reply:
That makes sense, works for me!